l3kernel/l3keys.dtx

```diff
@@ -3726,6 +3788,12 @@
     The~module~'#2'~does~not~have~a~key~called~'#1'.\\
     Check~that~you~have~spelled~the~key~name~correctly.
   }
+\msg_new:nnnn { keys } { unknown-expansion }
+  { The~value~expansion~'#1'~is~unknown. }
+  {
+    Key~values~can~only~be~expanded~using~one~of~the~pre-defined~methods:~
+    n,~o,~V~or~e.
```

Comment:
Lowercase-`v` is missing.

Reply:
By-design - see discussion with @FrankMittelbach about the logic of what seems reasonable to pas